metrics/lib/features/dashboard/presentation/model/project_metrics.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:math';
+
+import 'package:metrics/features/dashboard/presentation/model/build_result_bar_data.dart';
+
+/// Represents the metrics of the project.
+class ProjectMetrics {
```

Comment:
```suggestion
/// Represents the metrics of the project.
@immutable
class ProjectMetrics {
```
maybe we should go all the way to @immutable & const constructor?